app-games/src/main/java/com/aptoide/android/aptoidegames/appview/AppViewScreen.kt

```diff
@@ -208,6 +211,8 @@ fun AppViewContent(
   navigate: (String) -> Unit,
   navigateBack: () -> Unit,
 ) {
+  val (bundleTitle, bundleTag) = rememberBonusBundle() ?: (null to null)
```

Comment:
This looks weird.
Better us it without destructor like so:
`val bonusBundle = rememberBonusBundle()`